src/control/SConscript

```diff
@@ -204,5 +216,10 @@ def scons():
 
     AlwaysBuild([serverbin, adminbin])
 
+    if is_firmware_mgmt_build(denv):
+        fwbin = install_go_bin(senv, gosrc, [denv.nvmecontrol], "daos_firmware",
+                               "daos_firmware")
+        AlwaysBuild([fwbin])
```

Comment:
I would be fine with doing it in a follow up PR, but it occurs to me that we might want to emit a build message here to the effect that this is enabling experimental functionality. e.g. "Building EXPERIMENTAL daos_firmware helper" or something like that. Just to make people think twice about casually enabling and using it.

Reply:
That's a good idea. Can address it in a follow-on.